src/util/ws_message.py

```diff
@@ -13,7 +13,23 @@
 #           }
 
 
-def format_response(incoming_msg: Dict[str, Any], response_data: Dict[str, Any]) -> str:
+Request = TypedDict(
+    "Request", {"command": str, "data": Optional[Dict[str, Any]], "request_id": str, "destination": str, "origin": str}
```

Comment:
I think we can make data an `Optional[Union[CommandXDataType, CommandYDataType, CommandZDataType]]`, but maybe that is too much (I don't know).

Reply:
Yeah I think that's too much for now